pages/articles/category/[category]/index.tsx

```diff
@@ -181,7 +181,7 @@ const ArticlesPage: FC<Props> = props => {
                   <li key={i}>
                     <LinkButton
                       text={`${i + 1}`}
-                      href={i === 0 ? `/articles/category/${category}` : `/articles/category/${category}/page/${i + 1}`}
+                      href={i === 0 ? category === 'all' ?  '/articles' : `/articles/category/${category}` : `/articles/category/${category}/page/${i + 1}`}
```

Comment:
These ternary operators are getting out of hand. I think it's time to consider introducing a local function that creates a link out of a page number and a category. What do you think?